index.rst

```diff
@@ -496,6 +496,13 @@ What's new in PyTorch tutorials?
    :link: intermediate/torchserve_with_ipex
    :tags: Model-Optimization,Production
 
+.. customcarditem::
+   :header: Grokking PyTorch Intel CPU Performance from First Principles (Part 2)
+   :card_description: A case study on the TorchServe inference framework optimized with Intel® Extension for PyTorch (Part 2).
+   :image: _static/img/thumbnails/cropped/generic-pytorch-logo.png
+   :link: intermediate/torchserve_with_ipex_2
+   :tags: Model-Optimization,Production
```

Comment:
Hi @svekars, I see this tutorial has been published to pytorch.org [here](https://pytorch.org/tutorials/intermediate/torchserve_with_ipex_2.html?highlight=grokking%20pytorch%20intel%20cpu%20performance%20from%20first%20principles%20part). However, I don't see link to this tutorial in the left nav, could you pls. help check on this? 
![Screenshot (1374)](https://user-images.githubusercontent.com/93151422/196245046-b5cc89ae-3155-4ae2-a4f6-d7b81b01c903.png)